_sass/screen.scss

```diff
@@ -114,6 +114,17 @@ figure.code {
   }
 }
 
+figure.illustration {
+  margin: 1.5em 0;
+  img {
+    margin: 0;
+  }
+  figcaption {
+    font-size: 0.75em;
+    line-height: 1.5;
+  }
```

Comment:
I noticed we didn't have a pattern around captioning our illustrations, so I introduced a bit of new styling here. It's scoped to `figure.illustration` so that it doesn't conflict with the `figure.code` that gets applied to code snippets.